html-forms/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <div>
+    <label for="email">Email</label>
+    <form>
```

Comment:
You should not have a form element surrounding each input, all the inputs of the form should be contained in one `form` element.